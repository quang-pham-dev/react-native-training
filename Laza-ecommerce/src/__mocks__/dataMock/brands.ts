/* A data array. */

export const brand = {
  id: '0',
  name: 'Nike',
  logoUrl:
    'https://toppng.com/uploads/preview/ike-logo-png-hd-nike-logo-png-image-free-download-nike-logo-sv-11562865034fkhqkcoas6.png'
};

export const brands = [
  {
    id: '0',
    name: 'Nike',
    logoUrl:
      'https://toppng.com/uploads/preview/ike-logo-png-hd-nike-logo-png-image-free-download-nike-logo-sv-11562865034fkhqkcoas6.png'
  },
  {
    id: '1',
    name: 'Adidas',
    logoUrl:
      'https://www.pikpng.com/pngl/m/42-429866_adidas-logo-png-images-free-download-rh-pngimg.png'
  },
  {
    id: '2',
    name: 'Fila',
    logoUrl:
      'https://toppng.com/uploads/preview/fila-logo-vector-download-free-11574216707jcmvfu58fy.png'
  },
  {
    id: '3',
    name: 'Puma',
    logoUrl:
      'https://banner2.cleanpng.com/20180613/gsj/kisspng-tracksuit-puma-logo-brand-clothing-puma-logo-5b20dc6b232426.487874081528880235144.jpg'
  },
  {
    id: '4',
    name: 'Gucci',
    logoUrl: 'https://esquilo.io/png/thumb/XBwczcMncYeff65-Vector-Gucci-Logo-PNG-Free-Download.png'
  }
];
