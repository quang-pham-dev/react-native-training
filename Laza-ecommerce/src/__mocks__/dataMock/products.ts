import { ProductProps } from 'types/Products';

export const product: ProductProps = {
  id: '0',
  brandId: '1',
  name: 'Nike',
  price: 100,
  type: 'shoes',
  like: true,
  title: 'Nike Sportswear Club Fleece',
  source:
    'https://images.unsplash.com/photo-1542291026-7eec264c27ff?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80',
  description:
    "Nike is a multinaelichick and retired quarterback Jack Nicklaus. Nike's products include apparel, shoes, equipment, and accessories. Nike also owns and operates the majority of the world's sporting goods brands, including Adidas, Puma, and Fila.",
  reviewer: {
    id: '2',
    name: 'John Doe',
    date: '13, Sep, 2020',
    image:
      'https://images.unsplash.com/photo-1609505848912-b7c3b8b4beda?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=765&q=80',
  },
  imageReview: [
    {
      id: '0',
      image:
        'https://images.unsplash.com/photo-1499852848443-3004d6dc4cfc?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1374&q=80',
    },
    {
      id: '1',
      image:
        'https://images.unsplash.com/photo-1618898909019-010e4e234c55?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80',
    },
    {
      id: '2',
      image:
        'https://images.unsplash.com/photo-1575537302964-96cd47c06b1b?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
    },
    {
      id: '3',
      image:
        'https://images.unsplash.com/photo-1518656306295-aa28b28b2504?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
    },
    {
      id: '4',
      image:
        'https://images.unsplash.com/photo-1609505848912-b7c3b8b4beda?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=765&q=80',
    },
    {
      id: '5',
      image:
        'https://images.unsplash.com/photo-1575537302964-96cd47c06b1b?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
    },
  ],
  rating: '4.5',
  comment:
    'lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,adipisicing elit. Quisquam, quia.',
};

export const products: ProductProps[] = [
  {
    id: '0',
    brandId: '1',
    name: 'Nike',
    price: 100,
    type: 'shoes',
    like: true,
    title: 'Nike Sportswear Club Fleece',
    source:
      'https://images.unsplash.com/photo-1542291026-7eec264c27ff?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80',
    description:
      "Nike is a multinaelichick and retired quarterback Jack Nicklaus. Nike's products include apparel, shoes, equipment, and accessories. Nike also owns and operates the majority of the world's sporting goods brands, including Adidas, Puma, and Fila.",
    reviewer: {
      id: '2',
      name: 'John Doe',
      date: '13, Sep, 2020',
      image:
        'https://images.unsplash.com/photo-1609505848912-b7c3b8b4beda?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=765&q=80',
    },
    imageReview: [
      {
        id: '0',
        image:
          'https://images.unsplash.com/photo-1499852848443-3004d6dc4cfc?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1374&q=80',
      },
      {
        id: '1',
        image:
          'https://images.unsplash.com/photo-1618898909019-010e4e234c55?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80',
      },
      {
        id: '2',
        image:
          'https://images.unsplash.com/photo-1575537302964-96cd47c06b1b?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
      },
      {
        id: '3',
        image:
          'https://images.unsplash.com/photo-1518656306295-aa28b28b2504?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
      },
      {
        id: '4',
        image:
          'https://images.unsplash.com/photo-1609505848912-b7c3b8b4beda?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=765&q=80',
      },
      {
        id: '5',
        image:
          'https://images.unsplash.com/photo-1575537302964-96cd47c06b1b?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
      },
    ],
    rating: '4.5',
    comment:
      'lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,adipisicing elit. Quisquam, quia.',
  },
  {
    id: '0',
    brandId: '1',
    name: 'Nike',
    price: 100,
    type: 'shoes',
    like: true,
    title: 'Nike Sportswear Club Fleece',
    source:
      'https://images.unsplash.com/photo-1542291026-7eec264c27ff?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80',
    description:
      "Nike is a multinaelichick and retired quarterback Jack Nicklaus. Nike's products include apparel, shoes, equipment, and accessories. Nike also owns and operates the majority of the world's sporting goods brands, including Adidas, Puma, and Fila.",
    reviewer: {
      id: '2',
      name: 'John Doe',
      date: '13, Sep, 2020',
      image:
        'https://images.unsplash.com/photo-1609505848912-b7c3b8b4beda?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=765&q=80',
    },
    imageReview: [
      {
        id: '0',
        image:
          'https://images.unsplash.com/photo-1499852848443-3004d6dc4cfc?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1374&q=80',
      },
      {
        id: '1',
        image:
          'https://images.unsplash.com/photo-1618898909019-010e4e234c55?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80',
      },
      {
        id: '2',
        image:
          'https://images.unsplash.com/photo-1575537302964-96cd47c06b1b?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
      },
      {
        id: '3',
        image:
          'https://images.unsplash.com/photo-1518656306295-aa28b28b2504?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
      },
      {
        id: '4',
        image:
          'https://images.unsplash.com/photo-1609505848912-b7c3b8b4beda?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=765&q=80',
      },
      {
        id: '5',
        image:
          'https://images.unsplash.com/photo-1575537302964-96cd47c06b1b?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
      },
    ],
    rating: '4.5',
    comment:
      'lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,adipisicing elit. Quisquam, quia.',
  },
];
