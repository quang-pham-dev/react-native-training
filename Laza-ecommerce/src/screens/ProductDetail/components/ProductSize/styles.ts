import { StyleSheet } from 'react-native';

// Themes
import Fonts from 'themes/Fonts';
import Metrics from 'themes/Metrics';
import Colors from 'themes/Colors';

const styles = StyleSheet.create({
  sizeContainer: {
    width: '100%'
  },

  textSizeWrapper: {
    flexDirection: 'row',
    justifyContent: 'space-between'
  },

  textSizeGuide: {
    fontSize: Fonts.size.default,
    color: Colors.primaryColor
  },

  sizeWrapper: {
    width: '100%',
    marginTop: 10
  },

  sizeItem: {
    justifyContent: 'center',
    alignItems: 'center',
    width: 60,
    height: 60,
    borderRadius: Metrics.borderRadius.roundedMedium,
    backgroundColor: Colors.secondaryBackground,
    marginRight: 10
  }
});

export default styles;
