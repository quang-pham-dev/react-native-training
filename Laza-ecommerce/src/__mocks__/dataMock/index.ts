import { user } from './user';
import { brand, brands } from './brands';
import { product, products } from './products';

export { brand, brands, product, products, user };
