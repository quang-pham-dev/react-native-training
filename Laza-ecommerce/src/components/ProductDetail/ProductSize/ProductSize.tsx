import React from 'react';
import { PRODUCTS_SIZE } from 'constants/Products';
import { View, ScrollView, Pressable, Text } from 'react-native';
import styles from './styles';

export function ProductSize() {
  return (
    <View style={styles.sizeContainer}>
      <View style={styles.textSizeWrapper}>
        <Text style={styles.textSize}>Size</Text>
        <Text style={styles.textSizeGuide}>Size Guide</Text>
      </View>
      <ScrollView style={styles.sizeWrapper} horizontal showsHorizontalScrollIndicator={false}>
        {PRODUCTS_SIZE.map((item: string) => (
          <Pressable style={styles.sizeItem} key={item}>
            <Text style={styles.textSize}>{item}</Text>
          </Pressable>
        ))}
      </ScrollView>
    </View>
  );
}
