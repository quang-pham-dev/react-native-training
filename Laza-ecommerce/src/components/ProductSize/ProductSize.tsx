import React from 'react';
import { View, ScrollView, Pressable, Text } from 'react-native';

// Constants
import { PRODUCTS_SIZE } from 'constants/Products';

// Styles
import styles from './styles';

const ProductSize = () => {
  return (
    <View style={styles.sizeContainer}>
      <View style={styles.textSizeWrapper}>
        <Text style={styles.textSize}>Size</Text>
        <Text style={styles.textSizeGuide}>Size Guide</Text>
      </View>
      <ScrollView style={styles.sizeWrapper} horizontal showsHorizontalScrollIndicator={false}>
        {PRODUCTS_SIZE.map((item: string) => (
          <Pressable style={styles.sizeItem} key={item}>
            <Text style={styles.textSize}>{item}</Text>
          </Pressable>
        ))}
      </ScrollView>
    </View>
  );
};

export default ProductSize;
