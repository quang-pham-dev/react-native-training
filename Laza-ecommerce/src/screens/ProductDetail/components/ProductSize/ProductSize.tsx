import React, { memo } from 'react';
import { View, ScrollView, Pressable, Text } from 'react-native';

// LIBS
import isEqual from 'react-fast-compare';

// Constants
import { IProductDetailItemProps } from 'types/screens/ProductDetail';
import { ProductSizeProps } from 'types/models/Products';

// Styles
import styles from './styles';

const ProductSize = ({ product }: IProductDetailItemProps) => {
  const { size } = product || {};

  return (
    <View style={styles.sizeContainer}>
      <View style={styles.textSizeWrapper}>
        <Text style={styles.textSize}>Size</Text>
        <Text style={styles.textSizeGuide}>Size Guide</Text>
      </View>

      <ScrollView style={styles.sizeWrapper} horizontal showsHorizontalScrollIndicator={false}>
        {Boolean(size) &&
          size.map((item: ProductSizeProps) => (
            <Pressable style={styles.sizeItem} key={item.id}>
              <Text style={styles.textSize}>{item.size}</Text>
            </Pressable>
          ))}
      </ScrollView>
    </View>
  );
};

export default memo(ProductSize, isEqual);
