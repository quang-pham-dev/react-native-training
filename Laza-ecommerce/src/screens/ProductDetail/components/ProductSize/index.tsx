import React, { memo } from 'react';
import { View, ScrollView, Pressable, Text } from 'react-native';

// LIBS
import isEqual from 'react-fast-compare';

// Components
import Title from 'components/Title';

// Constants
import { IProductDetailItemProps } from 'types/screens/ProductDetail';

// Types
import { ProductSizeProps } from 'types/models/Products';

// Styles
import styles from './styles';

const ProductSize = ({ sizes }: IProductDetailItemProps) => {
  return (
    <View style={styles.sizeContainer}>
      <View style={styles.textSizeWrapper}>
        <Title.titleLeftSection titleName='Size' />
        <Title.titleRightSection style={styles.textSizeGuide}>Size Guide</Title.titleRightSection>
      </View>

      <ScrollView style={styles.sizeWrapper} horizontal showsHorizontalScrollIndicator={false}>
        {Boolean(sizes) &&
          sizes?.map((item: ProductSizeProps) => (
            <Pressable style={styles.sizeItem} key={item.id}>
              <Title titleName={item.size} />
            </Pressable>
          ))}
      </ScrollView>
    </View>
  );
};

export default memo(ProductSize, isEqual);
