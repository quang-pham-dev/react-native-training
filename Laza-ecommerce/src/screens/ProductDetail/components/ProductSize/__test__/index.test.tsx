import React from 'react';

// LIBS
import renderer from 'react-test-renderer';

// Components
import ProductSize from 'screens/ProductDetail/components/ProductSize';

// Mock data
import { product } from '__mocks__/dataMock';

describe('Product detail size', () => {
  const tree = renderer.create(<ProductSize product={product} />);

  test('should render correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
