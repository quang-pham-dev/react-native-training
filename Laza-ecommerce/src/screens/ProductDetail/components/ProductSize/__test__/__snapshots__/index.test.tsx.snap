// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product detail size should render correctly 1`] = `
<View
  style={
    Object {
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#000",
            "fontFamily": "Inter_500Medium",
            "fontSize": 17,
            "lineHeight": 16,
            "textTransform": "none",
          },
          Array [
            Object {
              "fontFamily": "Inter_500Medium",
              "fontSize": 17,
              "lineHeight": 16,
            },
            undefined,
          ],
          Object {
            "textAlign": "left",
          },
        ]
      }
    >
      Size
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#000",
            "fontFamily": "Inter_500Medium",
            "fontSize": 17,
            "lineHeight": 16,
            "textTransform": "none",
          },
          Array [
            Object {
              "color": "#8F959E",
              "fontFamily": "Inter_400Regular",
              "fontSize": 13,
              "lineHeight": 14,
            },
            Object {
              "color": "#3D93F8",
              "fontSize": 17,
            },
          ],
          Object {
            "textAlign": "left",
          },
        ]
      }
    />
  </View>
  <RCTScrollView
    horizontal={true}
    showsHorizontalScrollIndicator={false}
    style={
      Object {
        "marginTop": 10,
        "width": "100%",
      }
    }
  >
    <View>
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#F5F6FA",
            "borderRadius": 10,
            "height": 60,
            "justifyContent": "center",
            "marginRight": 10,
            "width": 60,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#000",
                "fontFamily": "Inter_500Medium",
                "fontSize": 17,
                "lineHeight": 16,
                "textTransform": "none",
              },
              undefined,
              Object {
                "textAlign": "left",
              },
            ]
          }
        >
          S
        </Text>
      </View>
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#F5F6FA",
            "borderRadius": 10,
            "height": 60,
            "justifyContent": "center",
            "marginRight": 10,
            "width": 60,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#000",
                "fontFamily": "Inter_500Medium",
                "fontSize": 17,
                "lineHeight": 16,
                "textTransform": "none",
              },
              undefined,
              Object {
                "textAlign": "left",
              },
            ]
          }
        >
          M
        </Text>
      </View>
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#F5F6FA",
            "borderRadius": 10,
            "height": 60,
            "justifyContent": "center",
            "marginRight": 10,
            "width": 60,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#000",
                "fontFamily": "Inter_500Medium",
                "fontSize": 17,
                "lineHeight": 16,
                "textTransform": "none",
              },
              undefined,
              Object {
                "textAlign": "left",
              },
            ]
          }
        >
          L
        </Text>
      </View>
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#F5F6FA",
            "borderRadius": 10,
            "height": 60,
            "justifyContent": "center",
            "marginRight": 10,
            "width": 60,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#000",
                "fontFamily": "Inter_500Medium",
                "fontSize": 17,
                "lineHeight": 16,
                "textTransform": "none",
              },
              undefined,
              Object {
                "textAlign": "left",
              },
            ]
          }
        >
          XL
        </Text>
      </View>
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#F5F6FA",
            "borderRadius": 10,
            "height": 60,
            "justifyContent": "center",
            "marginRight": 10,
            "width": 60,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#000",
                "fontFamily": "Inter_500Medium",
                "fontSize": 17,
                "lineHeight": 16,
                "textTransform": "none",
              },
              undefined,
              Object {
                "textAlign": "left",
              },
            ]
          }
        >
          2XL
        </Text>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
